### DISCUSSION TITLE
Welcome to DracOS_VENOMIZER Discussions!

### DISCUSSION THREAD

**Discussion by ekovegeance** [ekovegeance/DracOS_VENOMIZER#3] [Q&A] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ekovegeance** [comment]
DracOS GNU/Linux Remastered ( https://github.com/dracos-linux ) is the Linux operating system from Indonesian , open source is built based on Debian live project under the protection of the GNU General Public License v3.0. This operating system is one variant of Linux distributions, which is used to perform security testing (penetration testing). Dracos linux in Arm by hundreds hydraulic pentest, forensics and reverse engineering. Use a GUI-based tools-tools the software using the CLI (command line interface) and GUI (graphical user interface) to perform its operations. Now Dracos currently already up to version 3.1.5 with the code name "KUNTILANAK WITH REMASTERED".

**Comment by evsystem7** [comment, marked as answer]
DracOS GNU/Linux Remastered ( https://github.com/dracos-linux ) is the Linux operating system from Indonesian , open source is built based on Debian live project under the protection of the GNU General Public License v3.0. This operating system is one variant of Linux distributions, which is used to perform security testing (penetration testing). Dracos linux in Arm by hundreds hydraulic pentest, forensics and reverse engineering. Use a GUI-based tools-tools the software using the CLI (command line interface) and GUI (graphical user interface) to perform its operations. Now Dracos currently already up to version 3.1.5 with the code name "KUNTILANAK WITH REMASTERED".

**Comment by evsystem7** [comment]
what new dracos

**Reply by ekovegeance** [reply]
DracOS Venomizer 1.2.4 code name: Mr.Robot

**Comment by ekovegeance** [comment]
DracOS Venomizer 1.2.4 code name: Mr.Robot

**Comment by ekovegeance** [comment]
1.2.4
